lib/v2/devolverdigital/blog.js

```diff
@@ -6,36 +6,48 @@ module.exports = async (ctx) => {
 
     const { data: response } = await got(baseUrl);
     const $ = cheerio.load(response);
-    $('noscript').remove();
-
     const nextData = JSON.parse($('#__NEXT_DATA__').text());
-    const allBlogContents = $('div.flex > div > div > div > div:not([class])');
-
-    const items = nextData.props.pageProps.posts.map((post, postIndex) => {
-        // img resource redirection and
-        // clean up absolute layouts for img and span
-
-        const imageUrls = post.body.filter((item) => item.type === 'upload' && item.value.cloudinary.resource_type === 'image').map((item) => item.value.cloudinary.secure_url);
-
-        const allImageSpans = $(allBlogContents[postIndex]).find('span > img').parent();
-
-        allImageSpans.each((spanIndex, span) => {
-            $(span).attr('style', $(span).attr('style').replace('position:absolute', ''));
-            const img = $(span).find('img');
-            img.attr('src', imageUrls[spanIndex]);
-            img.attr('style', img.attr('style').replace('position:absolute', '').replace('width:0', '').replace('height:0', ''));
-        });
-
-        return {
-            title: post.title,
-            author: post.author,
-            pubDate: Date.parse(post.createdAt),
-            description: $(allBlogContents[postIndex]).html(),
-        };
-    });
+    const items = await Promise.all(
+        nextData.props.pageProps.posts.map(async (postData) => {
+            const postUrl = `${baseUrl}/post/${postData.id}`;
+            const postPage = await ctx.cache.tryGet(
+                postData.id + postData.updatedAt,
+                async () => {
+                    const { data: postPage } = await got(postUrl);
+                    return postPage;
+                },
+                60 * 60 * 24 * 30 // 30 days
```

Comment:
```suggestion
                },
```
The default cache time `CACHE_CONTENT_EXPIRE` is already enough.